src/containers/App.js

```diff
@@ -1,26 +1,9 @@
 import React from "react";
-import { Route, Switch, BrowserRouter, Link } from "react-router-dom";
-import IndexTypes from "./IndexTypes";
-import IndexCreature from "./IndexCreature";
+import Navigation from "../components/Navigation"
 
 const App = props => {
   return (
-    <BrowserRouter>
-      <div className="container">
-        <nav>
-          <ul>
-            <li>Navigation</li>
-            <li>
-              <Link to="/creatures">HOME</Link>
-            </li>
-          </ul>
-        </nav>
-        <Switch>
-          <Route exact path="/creatures" component={IndexTypes} />
-          <Route exact path="/creatures/:type" component={IndexCreature} />
-        </Switch>
-      </div>
-    </BrowserRouter>
+    <Navigation />
```

Comment:
This currently returns a empty component. There may be errors in your code due to this.